plugin/mutate/source/dextool/plugin/mutate/backend/database/standalone.d

```diff
@@ -2309,6 +2309,20 @@ struct DbCoverage {
         return rval;
     }
 
+    CovRegionStatus[] getCoverageStatus(FileId fileId) @trusted {
+        immutable sql = "SELECT t0.file_id, t0.begin, t0.end, t1.status FROM "~srcCovTable~" t0, "~srcCovInfoTable~" t1
```

Comment:
file_id is not used at line 2319-2320 thus can `t0.file_id` can be removed as a column in the result.